### PR TITLE
Updating Display of Uncertainties in the Examples

### PR DESCRIPTION
**Pull request by niazim3**
As mentioned in smiths/caseStudies#46, it would be preferred that the uncertainties of UncertainQuantities show up as a percent instead of a decimal between 0 and 1.

### CODE REVIEW

File: code/stable/nopcm/NoPCM_SRS.tex [hunk 1]
```diff
@@ -548,7 +548,7 @@ \section{Likely Changes}
 \item[\refstepcounter{lcnum}\lcthelcnum\label{LC:likeChg3}:]A\ref{A:assump9} - The temperature of the heating coil will actually change along its length as the water within it cools.
 \end{description}
 \begin{description}
-\item[\refstepcounter{lcnum}\lcthelcnum\label{LC:likeChg3}:]A\ref{A:assump9.npcm}- The model currently only accounts for charging of the tank. A more complete model would also account for discharging of the tank.
+\item[\refstepcounter{lcnum}\lcthelcnum\label{LC:likeChg3}:]A\ref{A:assump9.npcm}- The model currently only accounts for charging of the tank. That is, increasing the temperature of the water to match the temperature of the coil. A more complete model would also account for discharging of the tank.
```
Comment (niazim3) [inline code/stable/nopcm/NoPCM_SRS.tex]: @smiths Wanted to ensure the addition of the sentence here was alright. It's currently generated from Drasil, which is why copying over the contents from the built latex to the stable version added this sentence...

Comment (niazim3) [reply code/stable/nopcm/NoPCM_SRS.tex]: I'm confused as to why this sentence isn't showing up in the HTML file... looking into it now.

Comment (niazim3) [reply code/stable/nopcm/NoPCM_SRS.tex]: Update: as @szymczdm mentions in comment of commit 20537f7, the Travis Build should've failed earlier (when the .tex file wasn't updated for a change in that commit).

Nonetheless, this change is desired and just happened to occur in this pull request.

Comment (smiths) [reply code/stable/nopcm/NoPCM_SRS.tex]: @niazim3 the additional sentence is fine.

**Pull request merged**